.github/workflows/push_github_container_reg.yml

```diff
@@ -0,0 +1,38 @@
+name: Docker push
+# This builds the docker image and pushes it to GHCR
+# Runs on qbic-pipelines repo releases and push event to 'dev' branch (PR merges)
+on:
+  push:
+    branches:
+      - dev
+  release:
+    types: [published]
+
+jobs:
+  push_dockerhub:
+    name: Push new Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    # Only run for the qbicpipelines repo, for releases and merged PRs
+    if: ${{ github.repository == 'qbic-pipelines/rnadeseq' }}
+    steps:
+      - name: Check out pipeline code
+        uses: actions/checkout@v2
+
+      - name: Build new docker image
+        run: docker build --no-cache . -t ghcr.io/qbicpipelines/rnadeseq:latest
```

Comment:
Or `ghcr.io/qbicpipelines`?

Reply:
oops, that one is incorrect, well spotted!